Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ruby:2.3.3-onbuild
+RUN apt-get install -y curl
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
+RUN apt-get install -y nodejs mongodb
+RUN chmod +x /usr/src/app/start_putsbox.sh
```

Comment:
@dcohan nice! What's this `start_putsbox.sh`?

Reply:
@akash1233: The start_putsbox.sh doesn't exists anywhere?

@dcohan: I think it would be a good idea to have a specific "Docker" folder. Then you can easily remove it from any production servers. Ex: let Jenkins (CI tool) build a container. Then remove the "Docker" folder when copying the files to the container.